docs/cmake.md

```diff
@@ -0,0 +1,132 @@
+# CMake
+
+_Gears-Vk_ also supports building with CMake on Linux (*GCC* or *Clang*) and Windows (*MSVC*).
+
+There are currently no pre-built binaries of *Gears-Vk*'s dependencies for Linux included in the repository, so they are built alongside *Gears-Vk* the first time you build it.
+However, if there is already an *ASSIMP* build installed on your system (so that it can be found via `find_package(assimp)`), it will be used instead (e.g., on Ubuntu 20.04 `sudo apt-get install libassimp-dev` can be executed to get *ASSIMP* version 5.0.1).
+On Windows pre-built binaries of dependencies are included and used when building *Gears-Vk*.
+
+## CLion
+
+### Linux: Ubuntu
+
+No special settings are required on Linux: CLion can use the bundled cmake, you have to install the following packages on your system to get it work:
+```
+sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test
+sudo apt update && sudo apt install libassimp-dev g++-11 
+sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-11 60 --slave /usr/bin/g++ g++ /usr/bin/g++-11
+sudo add-apt-repository ppa:oibaf/graphics-drivers
+sudo apt update && sudo apt upgrade -y
+sudo apt install libvulkan-dev libvulkan1 mesa-vulkan-drivers vulkan-utils
+
+# Now verify your vulkan installation:
```

Comment:
```suggestion
# Verify your Vulkan installation:
```
^ Something's missing. Maybe like follows: "Verify the sanity of your Vulkan installation" or something like that?!